core/sr-primitives/src/weights.rs

```diff
@@ -16,26 +16,61 @@
 
 //! Primitives for transaction weighting.
 //!
-//! Each dispatch function within `decl_module!` can now have an optional
-//! `#[weight = $x]` attribute. $x can be any object that implements the
-//! `Weigh` trait. By default, All transactions are annotated by
-//! `#[weight = TransactionWeight::default()]`.
+//! Each dispatch function within `decl_module!` can have an optional `#[weight = $x]` attribute.
```

Comment:
Do we have a line length limit for comments? Rust standard seems to be 80.

Reply:
I usually do the same as code: 100. Is it wrong? Maybe module docs have a shorter limit.

Reply:
100 is fine, i think.